src/Sarif.WorkItems/SarifWorkItemModel.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.WorkItems;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.WorkItems
+{
+    public class SarifWorkItemModel : WorkItemModel<SarifWorkItemContext>
+    {
+        public void InitializeFromLog(SarifLog sarifLog, SarifWorkItemContext context = null)
+        {
+            this.Context = context ?? new SarifWorkItemContext();
+
+            context.InitializeFromLog(sarifLog);
+
+            // Shared GitHub/Azure DevOps concepts
+
+            this.LabelsOrTags = new List<string> { $"default{nameof(this.LabelsOrTags)}" };
+
+            // Note that we a simple file name here. The filers will add a prefix
+            // to the file name that includes other details, such as the id of
+            // the filed item.
+            this.Attachment = new Microsoft.WorkItems.Attachment
```

Comment:
>Microsoft.WorkItems [](start = 34, length = 19)

Ah, right, conflicts with SARIF `Attachment`.

Reply:
Yes, regrettable.

---
In reply to: [381024468](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381024468) [](ancestors = 381024468)